fixtures/context.html

```diff
@@ -1,11 +0,0 @@
-<!DOCTYPE html>
```

Comment:
I'm reluctant to pull this out. Are you absolutely sure this isn't used?

Reply:
This particular file was never in fixtures only this PR wants to add one (https://github.com/dequelabs/axe-test-fixtures/pull/5#pullrequestreview-734793327). All of the integrations have their own `context.html` outside the external directory, [playwright as an example](https://github.com/dequelabs/axe-core-npm/tree/develop/packages/playwright/tests/fixtures)